RATapi/examples/non_polarised/backgroundFunction.m

```diff
@@ -0,0 +1,14 @@
+function background = backgroundFunction(xdata,params)
+
+% Split up the params array....
+Ao = params(1);
+k = params(2);
+backConst = params(3);
+
+% Make an exponential decay background....
+background = zeros(numel(xdata),1);
+for i = 1:numel(xdata)
+    background(i) = Ao*exp(-k*xdata(i)) + backConst;
+end
```

Comment:
this is copied from the corresponding MATLAB example, I don't know enough MATLAB to know what the alternatives are! are you suggesting it could be broadcast or something?

Reply:
Ahh, sorry. I thought it was a python function. We should convert this to python in time, but let's leave it for now.

Reply:
I was under the impression that Python background functions weren't supported yet, hence why I left it in MATLAB

Reply:
That's right, yes. To be clear, I meant it should be coverted to a python function when they are supported.